### ISSUE TITLE
Echo action should not be regarded as a test "step"

### ISSUE THREAD

**Issue by Ernesto-Maserati-CH** [christophd/citrus#170]
I am surprised that "echo" is logged as a test step in the console output.

`16:28:24,379 DEBUG        citrus.Citrus| TEST STEP 4 SUCCESS`
`16:28:24,379 DEBUG  context.TestContext| Setting variable: foo with value: 'bar'`

It is convenient to use the echo method, but I don't regard it as a test "step" at all since it is just logging something.

Can you please turn off that it is displayed as a test step, or make that behavior configurable (preferably with default that it is not a test step)?

**Comment by christophd** [comment]
You are right. Sometimes the obvious just needs to be said by someone else. I hope we can do this as the step logging is done in a generic way with a test action listener. I will have a look at this

**Comment by olha-bastieieva** [comment]
It seems that my issue is related to this one, so I'm adding comment here. It would be nice also to have "createVariable" as not a step in logs as well.
And it would be great to have steps for actions inside parallel containers (right now we have one steps for whole container)

**Comment by svettwer** [comment]
Hi!

Knowing about variables that define some test data should remain a test step in my opinion. :thinking: 

BR,
Sven
